rollup.config.js

```diff
@@ -0,0 +1,19 @@
+import configGenerator from '@ovh-ux/component-rollup-config';
+
+const config = configGenerator({
+  input: './src/index.js',
+});
+
+export default [
+  config.cjs(),
+  config.es(),
+  config.umd({
+    output: {
+      globals: {
+        '@ovh-ux/translate-async-loader': 'translate-async-loader',
```

Comment:
```suggestion
        '@ovh-ux/ng-translate-async-loader': 'ngTranslateAsyncLoader',
```

Reply:
@antleblanc You should manage this change in another pull request. If he does it now, it means that all managers are already been bumped so he can use his module.

Reply:
@euhmeuh you can omit my previous comment if needed.

Reply:
Seen together. This dependency is no more needed.